.concourse/pipelines/master-commit.yaml

```diff
@@ -0,0 +1,193 @@
+                  aws_access_key_id: ((polymath-ecr.aws-access-id))
+                  aws_secret_access_key: ((polymath-ecr.aws-access-secret))
+                  aws_region: us-east-1
+              run:
+                path: bash
+                args: ["git-polymesh/.concourse/scripts/t-npm-install.sh", "git-polymesh", "npm-caches"]
+              outputs:
+                - name: npm-caches
+      - in_parallel:
+          - task: cargo-test
+            config:
+              inputs:
+                - name: git-polymesh
+                - name: build-caches
+              caches:
+                - path: git-polymesh/target
+              platform: linux
+              image_resource:
+                type: registry-image
+                source:
+                  repository: polymathnetwork/rust
+                  tag: debian-1.44.1
+                  aws_access_key_id: ((polymath-ecr.aws-access-id))
+                  aws_secret_access_key: ((polymath-ecr.aws-access-secret))
+                  aws_region: us-east-1
+              run:
+                path: bash
+                args: ["git-polymesh/.concourse/scripts/t-cargo-test.sh", "git-polymesh", "build-caches"]
+          - task: integration-test
+            config:
+              inputs:
+                - name: git-polymesh
+                - name: artifact
+                - name: npm-caches
+              platform: linux
+              image_resource:
+                type: registry-image
+                source:
+                  repository: polymathnetwork/node
+                  tag: 14
+                  aws_access_key_id: ((polymath-ecr.aws-access-id))
+                  aws_secret_access_key: ((polymath-ecr.aws-access-secret))
+                  aws_region: us-east-1
+              run:
+                path: bash
+                args: ["git-polymesh/.concourse/scripts/t-npm-test.sh", "git-polymesh", "npm-caches", "artifact"]
+      - in_parallel:
+          - put: s3-polymesh
+            params:
+              file: artifact/polymesh-*
+              acl: public-read
+          - put: dockerimage-polymesh
+            params:
+              build: artifact
+              dockerfile: artifact/Dockerfile.distroless
+              tag_file: artifact/tag_file
+              additional_tags: artifact/additional_tags.distroless
+              tag_prefix: distroless-
+          - put: dockerimage-polymesh
+            params:
+              build: artifact
+              dockerfile: artifact/Dockerfile.debian
+              tag_file: artifact/tag_file
+              additional_tags: artifact/additional_tags.debian
+              tag_prefix: debian-
+      - put: slack-polymesh
+        params:
+          mode: normal
+          alert_type: success
+          message_file: semver-git-polymesh/version
+          fail_if_message_file_missing: true
+    on_failure:
+      put: slack-polymesh
+      params:
+        mode: normal
+        alert_type: failed
+
+
```

Comment:
If the build finishes successfully on master, new docs should be generated and pushed to cf workers.
See sample flow https://github.com/PolymathNetwork/Polymesh/blob/master/.github/workflows/docs.yml

Reply:
I will add this to the phase 2 targets to prevent scope creep on this PR.